zebra-state/src/service/finalized_state.rs

```diff
@@ -572,6 +587,15 @@ impl FinalizedState {
     pub fn path(&self) -> &Path {
         self.db.path()
     }
+
+    /// Returns the stored `ValueBalance` for the best chain at the finalized tip height.
+    pub fn current_value_pool(&self) -> ValueBalance<NonNegative> {
+        let value_pool_cf = self.db.cf_handle("tip_chain_value_pool").unwrap();
+        let tip_height = &self.finalized_tip_height().unwrap();
+        self.db
+            .zs_get(value_pool_cf, tip_height)
```

Comment:
Can we just store the tip chain value pool, rather than storing the chain value pool for every height?

We discovered that adding and deleting height-based indexes can use a lot of RocksDB disk space:
https://discord.com/channels/676527656170160146/716086297210650634/874222240936390677

Also, it makes the code simpler:
* we don't have to deal with `None` heights in the empty state
* we don't have to delete the previous height's chain value pool

```suggestion
            .zs_get(value_pool_cf, ())
```

Reply:
Ok, yea, this is a lot better. I applied it at https://github.com/ZcashFoundation/zebra/pull/2599/commits/26b40d913974e1f4a8f74b53a9330127edb84567